src/protocols/rdp/channels/rail.c

```diff
@@ -88,14 +90,46 @@ static UINT guac_rdp_rail_complete_handshake(RailClientContext* rail) {
     if (status != CHANNEL_RC_OK)
         return status;
 
+    RAIL_CLIENT_STATUS_ORDER client_status = {
+        .flags = 0x00
+    };
+
+    /* Send client status */
+    status = rail->ClientInformation(rail, &client_status);
+    if (status != CHANNEL_RC_OK)
+        return status;
+
     RAIL_SYSPARAM_ORDER sysparam = {
+
+        .dragFullWindows = FALSE,
+
+        .highContrast = {
+            .flags = 0x7E,
+            .colorScheme = {
+                .string = NULL,
+                .length = 0
+            }
+        },
```

Comment:
High contrast has to be enabled?

Reply:
My understanding is that this value (`0x7E`) is not _enabling_ high contrast mode, but declaring that the RDP server should allow the user to select high contrast mode through standard means. I've added commit 7598f5a to split this value up into its corresponding flags, which should make things clearer.

The value `0x7E` is the standard value passed when connecting via RemoteApp. It is shown within the example PDU in Microsoft's RemoteApp spec, is the default within FreeRDP, and was the value that Guacamole sent prior to the migration to FreeRDP 2.0.0.

Omitting this doesn't appear to break anything, but it would be an uncomfortable inconsistency with known-good (and possibly expected) behavior.